### ISSUE TITLE
no types for image library?

### ISSUE THREAD

**Issue by shriram** [brownplt/pyret-lang#1404]
The following program
```
include image
text("hi", 10, "red")
```
results in an error saying the applicant, `text`, has type `Any`.

**Comment by blerner** [comment]
This is a duplicate of all the "we need types for the image library !" issues we've had over the years. We have a typed image library lingering in a PR, and is blocked on getting (a) the new modules to work, (b) agreeing on the new apis, and (c) figuring out what types we want for the stringly-typed existing library.
